### ISSUE TITLE
React Fragment short syntax not supported in stateless component

### ISSUE THREAD

**Issue by alex-fournier** [eslint/eslint#10988]
**Tell us about your environment**

* **ESLint Version:** `5.7.0`
* **Node Version:** `8.11.3`
* **npm Version:** `5.6.0`

**What parser (default, Babel-ESLint, etc.) are you using?**

Babel-ESLint

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
// the airbnb config includes react plugin with
// react/jsx-uses-react and no-unused-vars rules
{
  "extends": "airbnb",
  "parser": "babel-eslint",
  "env": {
    "browser": true
  },
  "settings": {
    "react": {
      "version": "16.5.2"
    }
  }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```js
import React from 'react';
export default ({ formatter, item }) => <>{formatter(item)}</>
```

```bash
eslint --ext .jsx,.js .
```

**What did you expect to happen?**

No ESLint error.

**What actually happened? Please include the actual, raw output from ESLint.**

 1:8  error  'React' is defined but never used  no-unused-vars

**Comment by platinumazure** [comment]
Hi @alex-fournier, thanks for the issue.

I think you need to enable the rule `react/jsx-uses-react`. Then everything should hopefully work. If not, though, the issue is probably with eslint-plugin-react, not core ESLint.

**Comment by alex-fournier** [comment]
Actually, my config extends airbnb's config and plugin `react` is activated as well as `react/jsx-uses-react` rule. I updated my config in the issue.

I'll ask the same question in eslint-plugin-react, thank you.

**Comment by platinumazure** [comment]
Looks like this was fixed on the eslint-plugin-react side. Please make sure you're using the latest version of that plugin (assuming it's been released).

I'm going to go ahead and close this as resolved. If anything else does come up here on the ESLint side, let us know and we can reopen if needed. Thanks!
